docker/importer/importer.py

```diff
@@ -31,6 +31,7 @@
 import osv
 
 DEFAULT_WORK_DIR = '/work'
+PUBLIC_LOGGING_BUCKET = 'public-import-logging'
```

Comment:
nit: `osv-public-import-logging` to make it clearer?

Reply:
Changed it to `osv-public-import-logs`